wireguard/build-wireguard-go.sh

```
@@ -95,7 +95,20 @@ function build_windows {
     if $IS_WIN_ARM64; then
         local arch="aarch64"
         export GOARCH=arm64
-        export CC="aarch64-w64-mingw32-cc"
+        if [[ "${MSYSTEM:-}" != "CLANGARM64" ]]; then
+            local clangarm64_inc clangarm64_lib clangarm64_path clangarm64_resdir clang_path clang_ver
+            clangarm64_path=$(cygpath -m /clangarm64)
+            clangarm64_inc=$(cygpath -m /clangarm64/include)
+            clangarm64_lib=$(cygpath -m /clangarm64/lib)
+            clang_ver=$(ls /clangarm64/lib/clang/ 2>/dev/null | sort -V | tail -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)